packages/tomls/src/omnibus/testnet_source_chains.toml

```diff
@@ -16,6 +20,7 @@ tokensRouterPackage = "reya-tokens:1.0.0@router"
 [var.initial_proxy_vars]
 wstethSalt = "wsteth-main"
 usdeSalt = "usde-main"
+susdeSalt = "susde-main"
```

Comment:
testnet this is susde-main-2

Reply:
this is for source chains. i already deployed them with this salt and used them in the bridge, so will keep it like this for now